sankeys/views.py

```
@@ -0,0 +1,83 @@
+            
+            # 3. 데이터가 없을 경우 404 응답
+            if not flow:
+                return Response({
+                    "company_name": company.company_name,
+                    "error": "해당 기업의 재무 데이터가 준비되지 않았습니다."
+                }, status=404)
+
+            serializer = FinancialFlowSerializer(flow)
+            return Response(serializer.data)
+
+        except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)